src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+    private Random random = new Random();
+
     public String getRandomColor() {
-        return null;
+        int index = random.nextInt(Color.values().length);
```

Comment:
All magic numbers in your code should be constants. In this case, the number of colors (`Color.values().length`) can be extracted as a constant field to improve code readability and maintainability.